vividus/src/main/java/org/vividus/runner/DeprecatedStepsReplacer.java

```diff
@@ -75,7 +75,7 @@ public final class DeprecatedStepsReplacer
     private static final String END_OF_STEP_LOOKAHEAD = "(?=\\h*(\\v|$))";
     private static final Pattern NESTED_STEPS_TABLE_PATTERN = Pattern.compile("(\\v|^).\\h*step\\h*.\\v");
     private static final Pattern COMPOSITE_DEPRECATION_NOTIFICATION
-            = Pattern.compile("!--\\s+WARNING: The step .* is deprecated and will be removed in .*");
+            = Pattern.compile("!--\\s+Deprecated: The step .* is deprecated and will be removed in .*");
```

Comment:
can we do this in upper case or case insensitive? it's just more visible this way

Reply:
> can we do this in upper case or case insensitive? it's just more visible this way

Added `CASE_INSENSITIVE` flags.

Reply:
> > can we do this in upper case or case insensitive? it's just more visible this way
> 
> Added `CASE_INSENSITIVE` flags.

But actually it doesn't make any sense to do  deprecation pattern case insensitive. 
I think in this place better to leave only the uppercase